swagger/v1/api_v1.json

```diff
@@ -3515,6 +3704,9 @@
               "sidebar_left",
               "sidebar_left_2",
               "sidebar_right",
+              "feed_first",
+              "feed_second",
+              "feed_third",
```

Comment:
Note for reviewers:
These are the main changes that are important for this PR, everything else is random changes done by SWAGGER to generate dummy data or some indentation changes.